### PR TITLE
User should have a Dashboard

### PR DESCRIPTION
**Pull request by amandagibson**
#### Description
As a User
In order to see all of my information in one place
I would like to have a Dashboard

Changes proposed in this pull request:
* render dashboard
* create navigation for logo

#### Screenshots:
Screenshot 2019-04-09 at 21 06 33
Screenshot 2019-04-09 at 21 06 22

### CODE REVIEW

File: cypress/integration/fetchProjectActivities.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ describe("User can fetch project activities data", () => {
 			.type("api_kitten")
 			.get("button")
 			.contains("Login")
+			.click()
+			.get('button[name="time_tracking"]')
 			.click();
 	});
```
Comment (perhegg) [inline cypress/integration/fetchProjectActivities.js]: file name should be .feature for consistency.

File: cypress/integration/user_will_see_dashboard_on_login.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+	it('user can navigate back to dashboard by clicking the logo', () => {
+		cy
+			.get('button[name="time_tracking"]').click()
+			.get('img[id="logo"]').click();
+		cy
+			.get('h1[name="header"]')
+			.should('contain','My Dashboard')
+	})
+})
```
Comment (perhegg) [inline cypress/integration/user_will_see_dashboard_on_login.js]: file name

File: cypress/integration/fetchProjectActivities.js (hunk 1 above)
Comment (amandagibson) [reply cypress/integration/fetchProjectActivities.js]: you're right! fixed!

File: cypress/integration/user_will_see_dashboard_on_login.js (hunk 2 above)
Comment (amandagibson) [reply cypress/integration/user_will_see_dashboard_on_login.js]: changed!

**Pull request merged**